libs/ballot-interpreter-nh/src/metadata.rs

```diff
@@ -199,10 +199,13 @@ pub fn compute_bits_from_bottom_timing_marks(
         _ => {
             return Err(BallotPageMetadataError::InvalidTimingMarkCount {
                 expected: 2,
-                actual: bottom_timing_marks
-                    .iter()
-                    .filter(|&mark| mark.is_some())
-                    .count(),
+                actual: vec![
+                    bottom_timing_marks.first().unwrap(),
+                    bottom_timing_marks.last().unwrap(),
+                ]
+                .iter()
+                .filter(|&mark| mark.is_some())
+                .count(),
```

Comment:
Maybe this instead? It's more verbose, but pretty clear and doesn't `unwrap`.

```rust
    match (bottom_timing_marks.first(), bottom_timing_marks.last()) {
        (Some(Some(_)), Some(Some(_))) => {
            // both timing marks are present
        }
        (Some(None), Some(None)) => {
            return Err(BallotPageMetadataError::InvalidTimingMarkCount {
                expected: 2,
                actual: 0,
            });
        }
        (Some(None), _) | (_, Some(None)) => {
            return Err(BallotPageMetadataError::InvalidTimingMarkCount {
                expected: 2,
                actual: 1,
            });
        }
        _ => unreachable!(),
    }
```

Reply:
Sure!